src/components/popup/popup.js

```diff
@@ -0,0 +1,26 @@
+import template from "./popup.html";
+import { MDCMenu } from "@material/menu";
+
+export class PopupComponent {
+  constructor(mountPoint) {
+    this.mountPoint = mountPoint;
+  }
+
+  querySelectors() {
+    this.menuRoot = document.querySelector("#menu");
```

Comment:
looks like you don't need id here
![image](https://user-images.githubusercontent.com/14272396/37257895-e1025d74-2577-11e8-839a-9b23593bf11e.png)